integration/stability/agent_stability_test.sh

```diff
@@ -15,12 +15,10 @@ cidir=$(dirname "$0")
 source "${cidir}/../../metrics/lib/common.bash"
 
 # Environment variables
-IMAGE="${IMAGE:-busybox}"
+IMAGE="${IMAGE:-docker.io/library/busybox:latest}"
```

Comment:
nit, maybe worth using quay.io/prometheus/busybox:latest to avoid the pull limit

Reply:
That would be preferrable, indeed, thanks @snir911!

Reply:
Shouldn't there be some shared file somewhere that points to the various repos, e.g. docker, quay, ... so that we don't reinvent the wheel in every single test file?